modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -10,267 +10,10 @@
-|`./dynamic-plugins/dist/pagerduty-backstage-plugin-backend-dynamic`
-
-`PAGERDUTY_API_BASE`
-
-`PAGERDUTY_CLIENT_ID`
-
-`PAGERDUTY_CLIENT_SECRET`
-
-`PAGERDUTY_SUBDOMAIN`
-
-
-|Pingidentity  |`https://npmjs.com/package/@backstage-community/plugin-catalog-backend-module-pingidentity/v/0.1.4[@backstage-community/plugin-catalog-backend-module-pingidentity]` |0.1.4 
-|`./dynamic-plugins/dist/backstage-community-plugin-catalog-backend-module-pingidentity-dynamic`
-
-
-|Scaffolder Relation Processor  |`https://npmjs.com/package/@backstage-community/plugin-catalog-backend-module-scaffolder-relation-processor/v/2.0.1[@backstage-community/plugin-catalog-backend-module-scaffolder-relation-processor]` |2.0.1 
-|`./dynamic-plugins/dist/backstage-community-plugin-catalog-backend-module-scaffolder-relation-processor-dynamic`
-
-
-|Security Insights  |`https://npmjs.com/package/@roadiehq/backstage-plugin-security-insights/v/2.4.0[@roadiehq/backstage-plugin-security-insights]` |2.4.0 
-|`./dynamic-plugins/dist/roadiehq-backstage-plugin-security-insights`
-
-
-|ServiceNow  |`https://npmjs.com/package/@backstage-community/plugin-scaffolder-backend-module-servicenow/v/2.2.1[@backstage-community/plugin-scaffolder-backend-module-servicenow]` |2.2.1 
-|`./dynamic-plugins/dist/backstage-community-plugin-scaffolder-backend-module-servicenow-dynamic`
-
-`SERVICENOW_BASE_URL`
-
-`SERVICENOW_USERNAME`
-
-`SERVICENOW_PASSWORD`
-
-
-|SonarQube  |`https://npmjs.com/package/@backstage-community/plugin-sonarqube/v/0.8.5[@backstage-community/plugin-sonarqube]` |0.8.5 
-|`./dynamic-plugins/dist/backstage-community-plugin-sonarqube`
-
-
-|SonarQube  |`https://npmjs.com/package/@backstage-community/plugin-sonarqube-backend/v/0.2.25[@backstage-community/plugin-sonarqube-backend]` |0.2.25 
-|`./dynamic-plugins/dist/backstage-community-plugin-sonarqube-backend-dynamic`
-
-`SONARQUBE_URL`
-
-`SONARQUBE_TOKEN`
-
-
-|SonarQube  |`https://npmjs.com/package/@backstage-community/scaffolder-backend-module-sonarqube/v/2.2.2[@backstage-community/scaffolder-backend-module-sonarqube]` |2.2.2 
-|`./dynamic-plugins/dist/backstage-community-scaffolder-backend-module-sonarqube-dynamic`
-
-
-|Tech Radar  |`https://npmjs.com/package/@backstage-community/plugin-tech-radar/v/1.0.0[@backstage-community/plugin-tech-radar]` |1.0.0 
-|`./dynamic-plugins/dist/backstage-community-plugin-tech-radar`
-
-
-|Tech Radar  |`https://npmjs.com/package/@backstage-community/plugin-tech-radar-backend/v/1.0.0[@backstage-community/plugin-tech-radar-backend]` |1.0.0 
-|`./dynamic-plugins/dist/backstage-community-plugin-tech-radar-backend-dynamic`
-
-`TECH_RADAR_DATA_URL`
-
-
-|Utils  |`https://npmjs.com/package/@roadiehq/scaffolder-backend-module-utils/v/2.0.2[@roadiehq/scaffolder-backend-module-utils]` |2.0.2 
-|`./dynamic-plugins/dist/roadiehq-scaffolder-backend-module-utils-dynamic`
-
-
+%%TABLE_CONTENT_3%%
 |===
```

Comment:
@nickboldt No output is generated for the community plugins.

Reply:
hmm. good catch. that's odd. will look later. BTW you'll get a faster response in slack than in GH where you have to wait hours for me to find the email notification under the 100 other email notifications I have from jira and gh. :D